en/micro-integrator/docs/use-cases/examples/template_examples/using-endpoint-templates.md

```diff
@@ -28,50 +28,117 @@ For example, let's say we have two default endpoints with following hypothetical
       <retriesBeforeSuspension>3</retriesBeforeSuspension>
       <retryDelay>0</retryDelay>
     </markForSuspension>
-  </default>
+  </address>
 </endpoint>
 ```
 
 We can see that these two endpoints have different set of error codes and different progression factors for suspension. Furthermore, the number of retries is different between them. By defining an endpoint template, these two endpoints can be converged to a generalized form. This is illustrated in the following:
 
 ```
-<template name="ep_template" xmlns="http://ws.apache.org/ns/synapse">
-  <parameter name="codes"/>
-  <parameter name="factor"/>
-  <parameter name="retries"/>
-  <endpoint name="$name">
-    <default>
-        <suspendOnFailure>
-          <errorCodes>$codes</errorCodes>
-          <progressionFactor>$factor</progressionFactor>
-        </suspendOnFailure>
-        <markForSuspension>
-          <retriesBeforeSuspension>$retries</retriesBeforeSuspension>
-          <retryDelay>0</retryDelay>
-        </markForSuspension>
-    </default>
-  </endpoint>
+<template xmlns="http://ws.apache.org/ns/synapse" name="ep_template">
+   <parameter name="name"/>
+   <parameter name="uri"/>
+   <parameter name="codes"/>
+   <parameter name="factor"/>
+   <parameter name="retries"/>
+   <endpoint name="$name">
+      <address uri="$uri">
+         <suspendOnFailure>
+            <errorCodes>$codes</errorCodes>
+            <progressionFactor>$factor</progressionFactor>
+         </suspendOnFailure>
+         <markForSuspension>
+            <retriesBeforeSuspension>$retries</retriesBeforeSuspension>
+            <retryDelay>1</retryDelay>
+         </markForSuspension>
+      </address>
+   </endpoint>
 </template>
 ```
 
 !!! Note
-    - The Endpoint template uses parameters as inputs. These parameters are then refered using the `$` prefix within the template. Unlike sequence templates, endpoint templates are always parameterized using `$` prefixed values (not XPath expressions). For example, a parameter named `codes` can be referred by `$codes`.
+    - The endpoint template uses parameters as inputs. These parameters are then refered using the `$` prefix within the template. Unlike sequence templates, endpoint templates are always parameterized using `$` prefixed values (not XPath expressions). For example, a parameter named `codes` can be referred by `$codes`.
     - `$name` and `$uri` are default parameters that a template can use anywhere within the endpoint template (usually as parameters for endpoint name and address attributes).
 
 Since we have a template defined, we can use template endpoints to create two concrete endpoint instances with different parameter values for this scenario. This is shown below.
```

Comment:
Do not use "we". Suggestion: "The template is now complete. Therefore, you can use template endpoints to create two concrete endpoint instances with different parameter values for this scenario as shown below.

Reply:
Done